app/views/overview/_virtual-machine-row.html

```diff
@@ -47,32 +47,46 @@ <h3>
   </div>
   <div class="list-pf-expansion collapse" ng-if="row.expanded" ng-class="{ in: row.expanded }">
     <div class="list-pf-container">
-      <div class="expanded-section resource-details">
-        <h3>Details</h3>
-        <div class="row">
-          <dl class="dl-horizontal col-md-6">
-            <dt>State:</dt>
-            <dd>
-              <div vm-state ovm="row.apiObject"></div>
-              <span class="bar-separated" ng-if="row.canStartOvm()">
-                <a href="" ng-click="row.startOvm()">Start</a>
-              </span>
-              <span class="bar-separated" ng-if="row.canRestartOvm()">
-                <a href="" ng-click="row.restartOvm()">Restart</a>
-              </span>
-              <span class="bar-separated" ng-if="row.canStopOvm()">
-                <a href="" ng-click="row.stopOvm()">Stop</a>
-              </span>
-            </dd>
-          </dl>
-          <dl class="dl-horizontal col-md-6">
-            <dt>Operating System:</dt>
-            <dd>{{row.apiObject.metadata.labels['kubevirt.io/os'] || '--'}}</dd>
+      <div class="row-expanded-top">
 
-            <dt>Uptime:</dt>
-            <dd>{{ row.apiObject._pod | vmPodUptime }}</dd>
-          </dl>
+        <div class="overview-pod-template">
```

Comment:
The use of pod-template classes here is semantically incorrect as this is not a pod template.  You can achieve the desired result with a much simpler markup structure and one that is semantically meaningful to describe the contents of the vm, but you'll need to do a little work to roll your own (details in other comments).  

This div is not needed.

Reply:
ok, decoupled from the pod-template again